ebuild-maintenance/git/text.xml

```diff
@@ -141,7 +141,10 @@ rebase</c>.</li>
 Whenever repoman is not used to commit, you need to manually verify all
 packages affected by the commit using <c>repoman full</c>. Since repoman
 is not aware of staged changes, please make sure that all files are included
-in the commit.
+in the commit. Also when not using repoman, you must perform a manual sign-off
+to the <uri link="https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin">
```

Comment:
One thing I'd change is to make it more verbose that people are supposed to read what they're signing before signing it.

Reply:
you mean something like "make sure you have read the actual certificate", or more something that underlines the legal side of signing a commit

Reply:
The former. We don't want to get into a legal debate.

Reply:
added a line